### PR TITLE
feat (DOCS-CODE-013): [show usage pattern] add code comment, align format output with other tutorials, and more

### PR DESCRIPTION
**Pull request by ferantivero**

### CODE REVIEW

File: src/protected-api-access-protected-api/README.md [hunk 1]
```diff
@@ -21,40 +24,19 @@ urlFragment: ms-identity-docs-code-csharp
...
+  "businessPhones": [],
+  "displayName": "Maui",
+  "givenName": null,
+  "jobTitle": "dev",
+  "mail": null,
+  "mobilePhone": null,
+  "officeLocation": null,
+  "preferredLanguage": null,
+  "surname": null,
+  "id": "cff40dac-17ea-4183-9caf-65f2ee90c562"
```
Comment (ckittel) [inline src/protected-api-access-protected-api/README.md:39]: <3  Thanks.  I saw this last week and was going to mention it and then quickly forgot about it.

**Review by ckittel — APPROVED** [review]
On point!  🥳

**Review by skabou — APPROVED** [review]
:shipit:

**Comment by skabou** [comment]
Lol oops

**Pull request merged**